Makefile

```diff
@@ -14,6 +14,12 @@ load-fixtures:
 	sleep 5
 	docker-compose exec db psql "postgres://postgres:toor@db:5432/bcda?sslmode=disable" -f /var/db/fixtures.sql
 
+load-beneficiaries:
```

Comment:
`load-beneficiaries` target is not tied into the build.  I don't think we want to explicitly have to call it, do we?  I think the intention was to incorporate this data load as part of `load-fixtures`.  My recommendation is to remove the `load-beneficiaries` target completely, and just add this line to the end of `load-fixtures`:

`docker-compose exec db psql "postgres://postgres:toor@db:5432/bcda?sslmode=disable" -f /var/db/synthetic_beneficiaries.sql
`

Reply:
Yep, I think that was the original intent but there was a conversation off line about adding a load-beneficiaries target to the makefile and then executing it later when we actually want to perform the db queries. Your suggestion absolutely makes sense if we want to just load it all now. It really all depends on what the team wants.

Reply:
Thats cool.  If thats the case, then I would think we would at least want to update the Makefile so that `docker-bootstrap` calls `load-benficiaries`.  @rnagle what do you think?

Reply:
oh I see what you're saying, that makes sense to me

Reply:
Not opposed to adding to `make docker-boostrap`, we'll definitely want it there eventually. But, it could be troublesome/confusing until we actually start using the data in our app. For instance, with this data loaded, running `make models` will produce an unused `models/beneficiaries.xo.go` file.